slider/beatmap.py

```diff
@@ -1680,6 +1683,31 @@ def _resolve_stacking_old(self, hit_objects, ar, cs):
 
         return hit_objects
 
+    def closest_hitobject(self, t):
+        """The hitobject with a ``time`` closest to ``t`` (in ms).
```

Comment:
Please add a full numpydoc style docstring like the other methods; this is used to generate the documentation.